web-console/src/lib/functions/ddl.ts

```diff
@@ -216,22 +246,17 @@ export const findBaseType = (type: ColumnType): ColumnType => {
  * @param value
  * @returns
  */
-export const parseValueSafe = (sqlType: ColumnType, value: string) => {
+export const parseCSVValue = (sqlType: ColumnType, value: string) => {
```

Comment:
why CSV?

Reply:
This function is used just to parse the value from a CSV row. Soon when I am refactoring to xgress via JSON rather than CSV this will no longer be used, so I decided to mark its purpose appropriately